<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimilarityView</class>
 <widget class="QWidget" name="SimilarityView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wtoolbar" native="true">
     <property name="styleSheet">
      <string notr="true">#wtoolbar {
	border: 0px;
	border-top: 4px solid rgb(143, 195, 227);
	border-left: 1px solid gray;
	border-right: 1px solid gray;
	border-top-left-radius: 4px;
	border-top-right-radius: 4px;
	background-color: rgba(206, 206, 206, 200);
}

#btClose::hover{ background-color: rgb(200, 255, 251);}
#btMinimize::hover{ background-color: rgb(200, 255, 251);}

#btClose {
	background-color: rgb(143, 195, 227);
	border: 0px;
	border-top-right-radius: 4px;
	border-bottom-right-radius: 4px;
}

#btMinimize {
	background-color: rgb(143, 195, 227);
	border: 0px;
	border-top-left-radius: 4px;
	border-bottom-left-radius: 4px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btClose">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wcontainer" native="true">
     <property name="styleSheet">
      <string notr="true">#wcontainer {

background-color: rgba(206, 206, 206, 200);
border-bottom-left-radius: 6px;
border-bottom-right-radius: 6px;
border-left: 1px solid gray;
border-right: 1px solid gray;
border-bottom: 1px solid gray;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="wsearch" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="wartist" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QListWidget" name="listWidgetArtist">
              <property name="styleSheet">
               <string notr="true">#listWidgetArtist {
	background-color: rgba(234, 234, 234, 242);
/*	background-color: transparent;*/
	border: 1px solid gray;
	border-radius: 4px;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="showDropIndicator" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_4" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QPushButton" name="btSearchArtist">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>30</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>&gt;&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="walbum" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QListWidget" name="listWidgetAlbum">
              <property name="styleSheet">
               <string notr="true">#listWidgetAlbum {
	background-color: transparent;
	/*border: 1px solid gray;*/
	border-radius: 4px;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="showDropIndicator" stdset="0">
               <bool>false</bool>
              </property>
              <property name="iconSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="spacing">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_6" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QPushButton" name="btSearchAlbum">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>30</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>&gt;&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QListWidget" name="listViewSimilar">
           <property name="styleSheet">
            <string notr="true">#listViewSimilar {
	background-color: rgba(234, 234, 234, 242);
	border: 1px solid gray;
	border-radius: 4px;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::NoDragDrop</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="movement">
            <enum>QListView::Static</enum>
           </property>
           <property name="flow">
            <enum>QListView::TopToBottom</enum>
           </property>
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="viewMode">
            <enum>QListView::ListMode</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
